README.md

```diff
@@ -1,41 +1,21 @@
-# ocean-prefilter
+# [`ocean-prefilter` module](https://app.viam.com/module/viam-labs/ocean-prefilter)
 
-Viam provides an `ocean-prefilter` model of the [vision service](https://docs.viam.com//services/vision) to find objects of interest in large bodies of water.
+This module implements the [`rdk:service:vision` API](https://docs.viam.com/ml/vision/#api) in an `ocean-prefilter` model for your machine to find objects of interest in large bodies of water.
 
-It works by finding the horizon, cropping the image to only see the water, and then divides the water into patches. These patches are turned into histograms, which are interpreted as probability distributions for their color/intensity values. A Kolmogorov-Smirnov test is then done on the before and after image, to determine if they are "close enough" to being the same scene, or if something new has entered the scene. 
+A machine configured with this module is equipped to:
+- Locate the horizon, crop the image to only include the water, and then dividing the water into patches.
+- Turn the resultingn patches into histograms, which are interpreted as probability distrubutions for their color/intensity values.
+- Conduct a Kolmogorov-Smirnov test on the histograms from before and after images and determine if the scenes are consistent or if a new element has entered.
```

Comment:
I like splitting this up to make this easier to follow but because the module is doing the work without requiring additional input from the user, I think this makes more sense:
```suggestion
When you configure a machine with this module, the module:
1. Locates the horizon, crops the image to only include the water, and then divides the water into patches.
2. Turns the resulting patches into histograms, which it interprets as probability distributions for their color/intensity values.
3 Conducts a Kolmogorov-Smirnov test on the histograms from before and after images and determines if the scenes are consistent or if a new element has appeared.
```